src/js/Parser.js

```diff
@@ -0,0 +1,92 @@
+import { Station } from './Station';
+import { SubwayLine } from './SubwayLine';
+import { Subway } from './Subway';
+
+export function parseToSubway(data) {
+    let currentData = data;
+
+    currentData = handleToArray(currentData);
+    currentData = handleToSubway(currentData);
+
+
+    return createSubway(currentData);
+} 
+
+function handleToArray(data) {
+    let array = data;
+    array = array.split('\n');
+    array = Array.from(array);
+
+    array = array.map(name => {
+        return name.trim();
+    });
+
+    return array;
+}
+
+function handleToSubway(data) {
```

Comment:
handleToSubway isn't like English phrase. buildSubwayModel is more like what you want